docs/examples/postponed_annotations_broken.py

```diff
@@ -3,11 +3,12 @@
 
 
 def this_is_broken():
-    # List is defined inside the function so is not in the module's
+    # Annotations are defined inside the function so is not in the module's
```

Comment:
Probably better to revert this, it was slightly clear to the layman.

Reply:
The reason it was changed is that `List` is removed by pyupgrade in 3.9+, so I added `Any` that shouldn't be removed in any case. Other option is to revert changes and add # dont-upgrade to the top of the file, but it might be unclear why there's no future versions in this example.

And the third way — rethink the whole example so it would fit both versions without confusion on any of them.

Reply:
Yes, I think better to import one of the pydantic types like `HttpUrl` inside the function?

Reply:
Done!